ec2utils/README.md

```diff
@@ -0,0 +1,14 @@
+ec2utils
+========
+
+A collection of utilities that makes managing images in Amazon EC2 easier.
+All utilities utilize the same configuration file, ec2utils.conf that contains
```

Comment:
https://github.com/SUSE/Enceladus/pull/8/files#diff-e6f13b1dd86ec7269495a20a7410fbbaR63 refers to `.ec2utils.conf`; here there's no dot-prefix.